baseapp/queryrouter.go

```diff
@@ -6,16 +6,12 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// QueryRouter provides queryables for each query path.
-type QueryRouter interface {
-	AddRoute(r string, h sdk.Querier) (rtr QueryRouter)
-	Route(path string) (h sdk.Querier)
-}
-
 type queryRouter struct {
 	routes map[string]sdk.Querier
 }
 
+var _ sdk.QueryRouter = NewQueryRouter()
```

Comment:
Does the `QueryRouter` implementation also want to live in `types` too?

Reply:
yes, for the reason mentioned in my previous comment